### ISSUE TITLE
Outlook.com / Hotmail Account Stopped Sync

### ISSUE THREAD

**Issue by surface2air** [nylas/N1#834]
Great email client! I have been using the Nylas N1 email client for nearly 2 months for my Outlook.com / Hotmail email account. However, the Nylas N1 has stopped its sync for this web mail service, and I am no longer receiving email. I can still retrieve Hotmail email at Outlook.com, but not through Nylas N1 as of December 15, 2015. 

I am using Nylas N1 on a 2015 MacBook Pro with the latest version of Mac OS X: El Capitan.

Thanks,

Eric,
Seattle, Washington

**Comment by mbilker** [comment]
Hey @surface2air, thanks for the report! This is a common issue, and you are not alone.

I cannot seem to find the specific issues, but this was reported on the Slack channels as well.

**Comment by bengotow** [comment]
Hey! Looks like there was a temporary issue on our side — looks like your Hotmail account is syncing properly again. If you run into anything, feel free to re-open!

**Comment by surface2air** [comment]
Hi Ben,

Thanks for the reply! Should I remove my Hotmail under managing accounts and then try adding it back?

I do not see the sync reinstate and update with newer messages. Everything shows up through December 15, but not since.

Thanks in advance!

Cheers,

Eric

Eric Gould . eztiki@hotmail.com . Tel (206) 310-9645 . Seattle, Washington

Date: Tue, 29 Dec 2015 13:29:55 -0800

**Comment by bengotow** [comment]
Oh hmm—that's strange... your new mail should appear automatically. Is it the eztiki@hotmail.com email address you're having trouble with?

**Comment by surface2air** [comment]
Hi Ben,

Yes, thanks.

Eric

Eric Gould . eztiki@hotmail.com . Tel (206) 310-9645 . Seattle, Washington
Date: Tue, 29 Dec 2015 14:40:51 -0800

**Comment by bengotow** [comment]
Hey! Sorry for the hassle—I relayed this back to our platform team and they discovered a problem specific to Hotmail that is preventing your new mail from appearing. They're working on a fix—stay tuned!

**Comment by surface2air** [comment]
Hi Ben,
Sounds good! Thanks for the reply!
I look forward to using N1 again. It's really great email client, and it's perfect for Outlook.com / Hotmail.
Have a great New Year in 2016, and I look forward to hear back soon.
Thanks!
Eric Gould . eztiki@hotmail.com . Tel (206) 310-9645 . Seattle, WA
Sent from Outlook Mobile




On Tue, Dec 29, 2015 at 3:29 PM -0800, "Ben Gotow" <notifications@github.com> wrote:





Hey! Sorry for the hassle—I relayed this back to our platform team and they discovered a problem specific to Hotmail that is preventing your new mail from appearing. They're working on a fix—stay tuned!

---

**Comment by areeberg** [comment]
Hello, 
I can´t access any email from my hotmail account. I can´t even sync, and the only message that I get is no threads to display. 
Help please.
Thank you

**Comment by lyoko17220** [comment]
Same problem here since 2 months

**Comment by RicardoEPRodrigues** [comment]
Having the same problem. The last new message if from January.

**Comment by s-nel** [comment]
Having the same problem with my outlook.com account. I've tried removing and re-adding the account in Nylas. It gets all the way up to 5/13, just 6 days ago, but can't get anything newer. Setting up a new account using [IMAP/SMTP](http://windows.microsoft.com/en-us/windows/outlook/send-receive-from-app) settings seems to be working, and I suggest it as a workaround. This issue may be related to security issues when I first started using Nylas. Live.com alerted me that there was a suspicious login attempt (Nylas) and it was blocked. I unblocked Nylas, but now it won't get new messages.
